src/tests/myFinalTests.spec.js

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { currentOrderOnPage, sortArray, compareTwoArrays } = require('./utils');
+
+['az', 'za', 'lohi', 'hilo'].forEach((sortingType) => {
```

Comment:
Nice 👍

Reply:
I would improve this a little

```
[
{
sortingTitle: 'Z to A'.
sortingType: 'az'
},
{
title:
sortingType:
},
{
title: 'Low to High',
sortingType: 'lohi'
},
{
title:
sortingType:
}
].forEach(({sortingTitle, sortingType}) => {
test(`Test 1 - Verify sort by ${sortingTitle}`, async ({ inventoryPage }) => {});
}
```

In this way you will have more readable test names in the report

Reply:
Updated